angular/src/app/modules/candidate/candidate-staff-list/candidate-staff-list.component.ts

```diff
@@ -165,8 +168,23 @@ export class CandidateStaffListComponent extends PagedListingComponentBase<Candi
   }
 
   onReqSeletedCandidate(entity: CandidateStaff) {
-    this.subs.add(
-      this._reqStaff.createRequestCV(this.requisitionStaffId, entity.id).subscribe(res => {
+    const dialogRef = this._dialog.open(ConfirmPresentForHr, {
+      showHeader: false,
+      width: '25%',
+      contentStyle: { 'background-color': 'rgba(242,245,245)', overflow: 'visible' },
+      baseZIndex: 10000,
+      data: entity
+    });
+    dialogRef.onClose.subscribe((ref: { presentForHr: boolean}) => {
+      if (ref) {
+        const payload : RequisitionPayload = { 
+          cvId: entity.id,
+          requestId: this.requisitionStaffId, 
+          currentRequestId: entity.requisitionInfos[0]?.id || null ,
+          presentForHr: ref.presentForHr.toString() 
+        }
+      this.subs.add(
+      this._reqStaff.createRequestCV(payload).subscribe(res => {
         this.isLoading = res.loading;
```

Comment:
`this.isLoading = res.loading;` kiểm tra lại đoạn code này, biến `isLoading == true` trước đó không?